test/test_transformers/test_track_group_to_product_id_mapping_transformer.py

```diff
@@ -0,0 +1,32 @@
+from pyspark_test import PySparkTest
+from tidal_per_transformers.transformers.aggregate_transformer import AggregateTransformer
+import tidal_per_transformers.transformers.utils.constants as c
+from tidal_per_transformers.transformers.track_group_to_product_id_mapping_transformer import \
+    TrackGroupToProductIdMappingByUserCountryTransformer
+
+
+class AggregateTransformerTest(PySparkTest):
```

Comment:
I guess we can rename to TrackGroupToProductIdMappingByUserCountryTransformerTest (long name tho)